lib/delayed/backend/shared_spec.rb

```diff
@@ -574,7 +574,6 @@ def create_job(opts = {})
       it 're-schedules jobs after failing' do
         worker.work_off
         @job.reload
-        expect(@job.error).to_not be_nil
```

Comment:
I think it's sufficiently covered. This check made sense to want to know that the exception is assigned, but it's no longer possible to check.